container-core/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java

```diff
@@ -745,10 +745,10 @@ void requireThatRequestIsTrackedInAccessLog() throws IOException, InterruptedExc
                 new ServerConfig.Builder(),
                 new ConnectorConfig.Builder().accessLog(
                         new ConnectorConfig.AccessLog.Builder()
-                                .contentPathPrefixes("/status:1")
-                                .contentPathPrefixes("/state/v1:0.001")),
+                                .contentPathPrefixes("/search:1:1k")
```

Comment:
In these examples the suffix is always a one-character, so the regex part could be replaced with a more strict and self-documenting [kMG]?

Reply:
See [6467827](https://github.com/vespa-engine/vespa/pull/31456/commits/64678270e5ac9ab3a9f2b6523b4c2fc80999c4d9)